hack/update-generated-device-plugin-dockerized.sh

```diff
@@ -19,7 +19,7 @@ set -o nounset
 set -o pipefail
 
 KUBE_ROOT="$(cd "$(dirname "${BASH_SOURCE}")/../" && pwd -P)"
-DEVICE_PLUGIN_ROOT="${KUBE_ROOT}/pkg/kubelet/apis/deviceplugin/v1alpha1/"
+DEVICE_PLUGIN_ROOT="${KUBE_ROOT}/pkg/kubelet/apis/deviceplugin/v1alpha2/"
```

Comment:
Any reason we want to change api directory from v1alpha1 to v1alpha2 instead of just bumping version definition to 0.2 in pkg/kubelet/apis/deviceplugin/v1alpha1/constants.go?

Reply:
Good point. i have bumped the version in constants file. Versioning is expected to stick to our API naming schema - <majorversion><phase><sub-version>

Reply:
I remember something about changing the package name will break backward compatibility.

Reply:
Found the CRI issue in question: #47012
```
The problem is caused by grpc-go, which requires same package name between client and server. If they are not same, then no API calls could success.
```

Reply:
I'm not sure if this is a problem for an alpha update.
@vishh @jiayingz what do you think ?

Reply:
Right. For an incompatible API change like this, we will break compatibility. We feel it is ok since we are in alpha and hopefully we can avoid upgrading issue by making the current nvidia gpu device plugin a system addon on nodes with gpus.